src/Language/PureScript/Docs/Render.hs

```diff
@@ -152,9 +152,14 @@ renderDeclaration (P.DataDeclaration dtype name args ctors) title =
   children = map renderCtor ctors
   -- TODO: Comments for data constructors?
   renderCtor (ctor', tys) =
-          let typeApp' = foldl P.TypeApp (P.TypeConstructor (P.Qualified Nothing ctor')) tys
-              childCode = renderType typeApp'
-          in  RenderedChildDeclaration (show ctor') Nothing childCode Nothing ChildDataConstructor
+    RenderedChildDeclaration (show ctor') Nothing Nothing (ChildDataConstructor ctorSignature ctorCode)
```

Comment:
I've been referring to how a data constructor appears in the data declaration as a "signature", eg `Just a`, `Right b`, `Tuple a b`. The overlap with the more common meaning of "signature" in this context is undesirable, but I've been unable to come up with a better alternative. Any ideas?